research-software-directory/RSD-as-a-service
# fix: unify Dockerfile filenames

# Fix docker-compose not finding Dockerfiles / unify Dockerfile filenames

Fixes #174. The issue that `docker-compose` does not detect lowercase `dockerfile`s has been described in [this issue](https://github.com/docker/compose/issues/9188) in the docker-compose repository. It was apparently fixed in version 2.3.0, but somehow regressed again in 2.3.3. I just tested it with the latest version 2.3.4, which works fine with lowercase `dockerfile` again. So a pull request is not strictly necessary anymore. However, since some Dockerfiles are uppercase, some lowercase, I think it still makes sense to update the filenames to be consistent throughout the repo.

Changes proposed in this pull request:

* rename all lowercase `dockerfile` to `Dockerfile`

How to test:

* tested with docker-compose version 2.3.3
* run `docker-compose build` should complete without errors

PR Checklist:
